api/server.js

```diff
@@ -92,6 +92,32 @@ router.post('/api/events/createEvent', (req, res) => {
         });
 });
 
+// PUT -- update an event event
+router.put('/api/events/updateEvent', (req, res) => {
+    global.connection.query('UPDATE BetterLinkedIn_sp20.PlannedEvents SET (EventName, EventTime, EventDescription, IndustryID, OrganizerID) VALUES (?, ?, ?, ?, ?)',
+        [req.body.eventName, new Date(req.body.eventTime), req.body.eventDesc, req.body.industryID, req.body.userID],
```

Comment:
I don't think the event organizer should ever change because it's just whoever created the event. Also I don't think anyone except the organizer should be able to edit the event. Could you add an if statement to check for that in the client side?

Reply:
made the edit here and will add an if statement in the client side